spec/src/main/asciidoc/config.asciidoc

```diff
@@ -0,0 +1,147 @@
+relate to application methods.
+An annotated application method parameter can be referred to using a positional index, starting at '1'. `<fully qualified class name>/<method name>/<parameter position>/<spec-defined annotation short name>/<annotation attribute name>`.
+----
+// For example, the '1' below refers to the first parameter:
+   com.acme.order.OrderProcessing/setSomething/1/SomeMicroProfileAnno/someAttribute=5
+----
+
+[[scopedOverriding]]
+==== Properties That Override for a Particular Scope.
+
+Various specifications consider overriding, either at particular annotation location, a particular class or
+service wide (treated as global). Overriding at different scopes should occur as follows: 
+
+The set of possible scopes is:
+
+- class - `<dotted-fully-package-qualified-classname>/<annotation-classname>/<parameter>`
+- package-prefix - no recommended pattern as yet
+- package - `<dotted-fully-qualified-package>/<annotation-classname>/<parameter>` (no application class or method name)
+- archive - no recommended pattern as yet
+- service - [`mp.service-shortname.`]`<annotation-or-classname>/<parameter>`
+- globally - same as service
+- named - where an annotation occurrence is named - no recommended pattern as yet
+
+[[overridingPrecedence]]
+==== Overlapping Configuration Scopes Precedence
+
+Configuration which is more specific overrides configuration where the scope is more general.
+
+For example, where multiple configuration properties are present, the property `<classname>/<methodname>/<annotation>/<parameter>` takes precedence over `<classname>/<annotation>/<parameter>`, which would take precedence over `<annotation>/<parameter>`.
+
+[[codeEnvPrecedence]]
+==== Code Versus Process Environment Configuration Precedence 
+
+Configuration from the environment overrides configuration in the code for all configuration of equal of less specific scope in the code. 
+For example, the property `<classname>/<methodname>/<annotation>/<parameter>` takes precedence over `<classname>/<annotation>/<parameter>`, which is followed by `<annotation>/<parameter>`.
+
+----
+Should code sited configuration that is more specific than process environment configuration take precedence? Or should environment configuration overrule coded configuration for all scopes it applies to?
+----
+
+[[ignoredProperties]]
+==== Overrides Are Ignored if No Matching Annotation Is Present
+
+Overrides change the value of the corresponding annotation parameter specified in a microservice. If no annotation is present which matches the specified configuration property, the property will be ignored.
+
+[[empty]]
+==== Indicating An Empty Set [Move to Config Spec]
+
+(This topic is likely to be handled in a future version of the Configuration specification.)
+In order to guarantee that empty string configuration values are interpreted consistently, the following
+configuration values, as defined by the Configuration specification indicating empty:
```

Comment:
The verb is missing.  Possibly " indicating" should be ", indicate" ?

Reply:
I have changed it (next commit to come). My hope is to delete this section as it belongs in the Config Specification, it was only there as it was mentioned in a third spec.